ClinGen/clincoded
# Connects to #1471. Connects to #1668. Connects to #1670. Connects to #1671. Connects to #1680. Connects to #1681.

**Steps to test #1471:**
1. In a Gene-Disease record, start a new **Experimental** curation.
2. Try selecting each of 6 different experimental types and expect to see the **Score** dropdown menu at the bottom being enabled regardless whether the following **checkboxes** are selected or not:
"Has gene been implicated in same disease?" in _**Biochemical Function** A_
"Has this gene or genes been implicated in same disease?" in _**Protein Interactions**_
"Is the gene normally expressed in above tissues?" in _**Expression** A_
"Is expression altered in patients who have disease?" in _**Expression** B_
"Does wild-type rescue rescue the above phenotype?" in _**Rescue** (all 4 options)_ 
3. Try saving each of the above curations (with or without scoring) and expect to see the checkboxes' logic to continue to work as expected in regards to be the **required** fields.
4. Also expect to see a new yellow warning box below each of the checkboxes.

**Steps to test #1668:**
1. In a Gene-Disease record, curate/score 1 or 2 evidence.
2. Then go to the Classification Matrix and save it without any modified classification.
3. Subsequently, provision/approve the saved classification.
4. In the snapshot(s), expect to see the **Saved Classification** value being the "auto-calculated" value based on your scoring, instead of a "No Modification (modified)" text string.

**Steps to test #1670 & #1671:**
1. In the VCI, start the curation by entering **CA14274637** (a intergenic variant).
2. Proceed to the **Basic Info** tab and expect to see the "Unable to return RefSeq transcripts for this allele. Search RefSeq for transcripts." message in the **RefSeq Transcripts** panel and a similar message in the **Ensembl Transcripts** panel.
3. Proceed to the **Population** tab and expect to see the **gnomAD** table showing **Genome** population data.
4. Proceed to the **Gene-centric** tab and expect to see the panel content being rendered as expected without showing the _loading_ spinning wheels.

**Steps to test #1680:**
1. Sign into the curation interface and assign yourself to affiliation 10005 and/or 10006.
2. Log out and sign in again.
3. From the affiliation selection modal, expect to see the available affiliation names are **ClinGen Epilepsy Expert Panel** and/or **ClinGen Intellectual Disability & Autism Expert Panel** (instead of the "Working Group" text).

**Steps to test #1681:**
1. In the VCI, start the curation by entering a **55962** ClinVar ID.
2. Proceed to the **Variant Type** and the **MIssense** subtab.
3. In the **ClinGen Predictors** table, expect to see **>0.75** in the **Impact Threshold** column.
